tests/keras/preprocessing/text_test.py

```diff
@@ -49,6 +49,25 @@ def test_tokenizer():
         matrix = tokenizer.texts_to_matrix(texts, mode)
 
 
+def test_sequential_fit():
+    texts = ['The cat sat on the mat.',
+             'The dog sat on the log.',
+             'Dogs and cats living together.']
+    word_sequences = [
+        ['The', 'cat', 'is', 'dumb'],
```

Comment:
No value judgements in the unit tests, please

Reply:
I updated this with a more neutral sentiment.